readme.md

```diff
@@ -107,14 +114,21 @@ In this microbenchmark we compare two cases:
 
 * serializing an object after an object's property has been set to `undefined`
 * serializing an object after `delete` has been used to remove an object's property
+* serializing an object after `delete` has been used to remove an object's property that
+  was the last property added.
```

Comment:
serializing an object after `delete` has been used to remove the objects most recently added property.